### DISCUSSION TITLE
Plugin Name?

### DISCUSSION THREAD

**Discussion by tgrosinger** [tgrosinger/slated-obsidian#6] [Q&A]
Does anyone have a suggestion for the name of this plugin?

It was originally called "Agenda" however I renamed it to "Scheduled" to reflect the shifting of functionality towards task repetition and rescheduling.

Let me know if you have any other ideas!

**Comment by ericaxu** [comment]
Do you want a descriptive or a catchy name?

If you want something catchy (and potentially a little silly), "Obsidian Taskforce" just popped into my head. 😄

**Reply by ericaxu** [reply]
May the force be with you 🙏

**Comment by liamcain** [comment]
I think it could warrant a more "fun" name! Something like 
- Daily Grind
- Tasked
- Assistant
- TODO Tada
- Find the time!

Hmm, not feeling too creative. Will come up with more later.

**Comment by tgrosinger** [comment]
How about Plainer Planner? As in, a simplified planner? I do like Taskforce as well.

**Comment by sambbaron** [comment]
You could play off the Obsidian "stone" theme. Task Tablet?

**Reply by liamcain** [reply]
Slated!

**Reply by sambbaron** [reply]
Oh wow, that's really good

**Reply by tgrosinger** [reply]
I think it's decided. Thanks @liamcain!
